From 1f8252470ce43dc8a0680871013e2f4492764302 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Mon, 28 Feb 2022 10:27:09 -0500
Subject: [PATCH] data: Enable logo extension out of the box

Our brand team would like the logo extension to be used on new
installs.

This commit makes sure it gets enabled out of the box.
---
 data/org.gnome.shell.gschema.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.shell.gschema.xml.in b/data/org.gnome.shell.gschema.xml.in
index d5ea1e3..e3f440c 100644
--- a/data/org.gnome.shell.gschema.xml.in
+++ b/data/org.gnome.shell.gschema.xml.in
@@ -1,45 +1,45 @@
 <schemalist>
   <schema id="org.gnome.shell" path="/org/gnome/shell/"
           gettext-domain="@GETTEXT_PACKAGE@">
     <key name="development-tools" type="b">
       <default>true</default>
       <summary>
         Enable internal tools useful for developers and testers from Alt-F2
       </summary>
       <description>
         Allows access to internal debugging and monitoring tools
         using the Alt-F2 dialog.
       </description>
     </key>
     <key name="enabled-extensions" type="as">
-      <default>[]</default>
+      <default>['background-logo@fedorahosted.org']</default>
       <summary>UUIDs of extensions to enable</summary>
       <description>
         GNOME Shell extensions have a UUID property; this key lists extensions
         which should be loaded. Any extension that wants to be loaded needs
         to be in this list. You can also manipulate this list with the
         EnableExtension and DisableExtension D-Bus methods on org.gnome.Shell.
       </description>
     </key>
     <key name="disabled-extensions" type="as">
       <default>[]</default>
       <summary>UUIDs of extensions to force disabling</summary>
       <description>
         GNOME Shell extensions have a UUID property; this key lists extensions
         which should be disabled, even if loaded as part of the current mode.
         You can also manipulate this list with the EnableExtension and
         DisableExtension D-Bus methods on org.gnome.Shell.
         This key takes precedence over the “enabled-extensions” setting.
       </description>
     </key>
     <key name="disable-user-extensions" type="b">
       <default>false</default>
       <summary>Disable user extensions</summary>
       <description>
         Disable all extensions the user has enabled without affecting
         the “enabled-extension” setting.
       </description>
     </key>
     <key name="disable-extension-version-validation" type="b">
       <default>false</default>
       <summary>Disables the validation of extension version compatibility</summary>
-- 
2.35.1

